recipes/msdf-atlas-gen/all/conanfile.py

```diff
@@ -0,0 +1,84 @@
+import os
+from conans import ConanFile, CMake, tools
+
+required_conan_version = ">=1.33.0"
+
+
+class MsdfAtlasGenConan(ConanFile):
+    name = "msdf-atlas-gen"
+    license = "MIT"
+    homepage = "https://github.com/Chlumsky/msdf-atlas-gen"
+    url = "https://github.com/conan-io/conan-center-index"
+    description = "MSDF font atlas generator"
+    topics = ("conan", "msdf-atlas-gen", "msdf", "font", "atlas")
+    settings = "os", "compiler", "arch"
+    options = {"fPIC": [True, False]}
+    default_options = {"fPIC": True}
+
+    generators = "cmake"
+    exports_sources = ["CMakeLists.txt"]
+    _cmake = None
+
+    @property
+    def _source_subfolder(self):
+        return "source_subfolder"
+
+    def config_options(self):
+        if self.settings.os == "Windows":
+            del self.options.fPIC
+
+    def requirements(self):
+        self.requires("artery-font-format/1.0")
+        self.requires("msdfgen/1.9.1")
+
+    def validate(self):
+        if self.settings.compiler.get_safe("cppstd"):
+            tools.check_min_cppstd(self, 11)
+
+    def source(self):
```

Comment:
This is a binary-only recipe, so the actual compiler does not matter.
```suggestion
            tools.check_min_cppstd(self, 11)

    def package_id(self):
        del self.info.settings.compiler

    def source(self):
```